docs/userguide/mng-storage-driver.md

```diff
@@ -0,0 +1,145 @@
+<!--[metadata]>
++++
+title = "Choose a storage driver"
+description = "Learn how select the proper storage driver for your container."
+keywords = ["container, storage, driver, AUFS, btfs, devicemapper,zvfs"]
+[menu.main]
+parent = "mn_use_docker"
++++
+<![end-metadata]-->
+
+# Choose a storage driver
+
+When you start the Docker `daemon`, you can specify which storage driver you want to use with your containers. A container's storage driver can affect the performance a containerized applications. So it is important to understand the different storage options available and select the right one for your application.
+
+
+## Understand the copy-on-write strategy
+
+Sharing is a good way to optimize resources. People do this instinctively in
+daily life. For example, twins Jane and Joseph taking an Algebra class at
+different times from different teachers can share the same exercise book by
+passing it between each other. Now, suppose Jane gets an assignment to complete
+the homework on page 11 in the book. At that point, Jane can make her own copy
+of page 11, complete the homework, and hand in her copy. The original book is unchanged.
+
+Computer programming developed a similar sharing mechanism for systems called
+the *copy-on-write* (COW) strategy. In this strategy, system processes that use
+the same data share the same instance of that data rather than having their own
+copy. At some point, if one process needs to modify or write to the data, only
+then does the operating system make a copy of the data for that process to use.
+Only the process that needs to write has access to the data copy. All the
+other processes continue to use the original data.
+
+Linux operating systems use the [COW strategy in many
+ways](https://en.wikipedia.org/wiki/Copy-on-write). For example, the  `fork()`
+and `clone()` system calls use a COW strategy when creating processes. Virtual
+memory management and disk storage snapshots also use COW strategies.
+
+## Docker and COW
+
+When you run a Docker container and load an image, Docker uses a copy-on-write
+strategy to optimize the container performance.  For example, consider this
+command line that runs a Python image:
+
+```bash
+$ docker run -ti python bash
+```
+
+If the container changes the `python` image, Docker only tracks (or writes) the
+changes between the container's image and the original image. The original
+`python` image is not modified and any other container also using the `python`
+image are unaffected.
+
+Docker's COW strategy saves space in the container because it need not have a
+whole image locally, only its changes.  And this strategy reduces the time to
+start a container because it downloads only the part of an image it needs to
+run.
```

Comment:
not sure this is fully correct; or at least confusing, because docker will download all layers of an image (on docker pull), and also needs all layers to start the container, just changes in the running container will be in a writable layer

(hope I make sense here)